src/index.tsx

```diff
@@ -1,13 +1,33 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
 
-createRoot(document.getElementById('root') as HTMLDivElement).render(
+export const Root = () => (
   <Router>
-    <App />
-  </Router>,
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="*" element={<NotFoundPage />} />
+        <Route index element={<HomePage />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route index element={<PeoplePage />} />
+          <Route path=":personId" element={<PeoplePage />} />
```

Comment:
The nested `Route` for `PeoplePage` with `index` and `:personId` paths seems redundant, as both render the same `PeoplePage` component. Consider differentiating the components or paths if they are intended to display different content.